vendor/k8s.io/kubernetes/staging/src/k8s.io/apiserver/pkg/util/proxy/dial_test.go

```diff
@@ -96,7 +96,7 @@ func TestDialURL(t *testing.T) {
 			ts.TLS = &tls.Config{Certificates: []tls.Certificate{cert}}
 			ts.StartTLS()
 
-			tlsConfigCopy := utilnet.CloneTLSConfig(tc.TLSConfig)
+			tlsConfigCopy := tc.TLSConfig
```

Comment:
Arrgh, that wouldn't work, since utilnet.CloneTLSConfig wouldn't trigger the call to serverInit in tlsConfigCopy.

@liggitt I'm not sure how to make this test work for both 1.7 and 1.8, unless we selectively use Clone for golang 1.8 and utilnet.CloneTLSConfig for 1.7, but I'm not sure if that leaves us open for unexpected behavior under golang 1.8.